demos/2_Iterative/03_PDHG.ipynb

```diff
@@ -289,7 +288,10 @@
    "id": "dc945eb5-17d9-476e-b4dd-998b9b90d51e",
    "metadata": {},
    "source": [
-    "### Slice through projections"
+    "### Slice through projections\n",
+    "\n",
+    "#### Warning\n",
+    "Notice that calling `islicer` will create another reference to the 3D data, therefore even if you `del data3D` Python will not clear the memory it points to. If you are short of memory, running `islicer` and deleting `data3D` will not have any effect on your memory usage."
```

Comment:
```suggestion
    "Notice that calling `islicer` will create another reference to the 3D data, therefore even if you `del data3D` Python will not clear the memory it points to. If you are short of memory, we recommend not to run `islicer`, because running it and deleting `data3D` will not have any effect on your memory usage."
```